src/components/Home.js

```diff
@@ -1,10 +1,26 @@
-import React from 'react';
 
```

Comment:
Files should not begin with an empty line

Reply:
Oh, I will fix it.